Plutarch/Bitwise.hs

```diff
@@ -21,39 +19,6 @@ import Plutarch.Integer (PInteger)
 import Plutarch.Unsafe (punsafeBuiltin)
 import PlutusCore qualified as PLC
 
-{-
-CONVERSION
--}
-
-{- | Convert a 'PInteger' into a 'PByteString', as described in
-[CIP-121](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0121).
-The first argument indicates the endianness of the conversion and the third
-argument is the integer to be converted, which must be non-negative.  The
-second argument must also be non-negative and it indicates the required width
-of the output.  If the width is zero then the output is the smallest
-bytestring which can contain the converted input (and in this case, the
-integer 0 encodes to the empty bytestring).  If the width is nonzero then the
-output bytestring will be padded to the required width with 0x00 bytes (on
-the left for big-endian conversions and on the right for little-endian
-conversions); if the input integer is too big to fit into a bytestring of the
```

Comment:
Documentation about which side the padding will go is gone. Though it's kind of obvious, I think it's good to have.